index.html

```diff
@@ -185,7 +217,7 @@ <h4>FEC website</h4>
           Although the dates of FEC’s traffic spikes may be predictable, it’s still hard to prepare for the extra demands on the system. “A server that’s still operable but near the end of its life can have unexpected problems” and buying a new server can take months. But on cloud.gov, “this isn’t our problem.”
         </p>
         <p>
-          <a href="{{ site.baseurl }}/docs/customer-stories/fec/" class="cg-arrow">Read the FEC case study</a>
+          <a href="/docs/customer-stories/fec/" class="cg-arrow">Read the FEC case study</a>
```

Comment:
Why the change? Generally we want to use `{{ site.baseurl }}` as it is the recommended way to create links in Jekyll